apps/system/js/nfc_handover_manager.js

```diff
@@ -153,37 +135,15 @@
      * @memberof NfcHandoverManager.prototype
      */
     _start: function _start() {
-      var self = this;
-
-      this.bluetooth = navigator.mozBluetooth;
-
       this.incomingFileTransferInProgress = false;
       this.bluetoothStatusSaved = false;
       this.bluetoothAutoEnabled = false;
 
-      if (typeof(window.navigator.mozBluetooth.onattributechanged) !==
-        'undefined') {
-        this._debug('Bluetooth APIv2 does not support NFC yet');
-        return;
-      }
-      if (this.bluetooth.enabled) {
-        this._debug('Bluetooth already enabled on boot');
-        var req = this.bluetooth.getDefaultAdapter();
-        req.onsuccess = function bt_getAdapterSuccess() {
-          self.defaultAdapter = req.result;
-          self._debug('MAC address: ' + self.defaultAdapter.address);
-          self._debug('MAC name: ' + self.defaultAdapter.name);
-        };
-        req.onerror = function bt_getAdapterError() {
-          self._logVisibly('init: Failed to get bluetooth adapter');
-        };
-      }
-
```

Comment:
the adapter retrieving now counts on `Service.query`, these code are  not necessary any more